gutenTAG/visualization.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import os
+import pathlib
+
+PROJECT_DIR = pathlib.Path().resolve()
+sys.path.append(str(PROJECT_DIR))
+
+import pandas as pd
+import plotly.graph_objs as go
+
+"""
+This visualization will only work with one or two value columns.
+It is a small test to show what's going on.
+"""
```

Comment:
I don't see a reason to include this script into the GutenTAG code base.

- It's not robust. It does not work for all possible GutenTAG time series.
- It's neither integrated into the GutenTAG CLI nor into the GutenTAG API.
- Plotting all time series for large collections is computationally infeasible.
- GutenTAG already has plotting functionality `gutenTAG --plot` (while generating the time series)